src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/SunPKCS11.java

```diff
@@ -120,7 +120,7 @@ public final class SunPKCS11 extends AuthProvider {
 
     // This is the SunPKCS11 provider instance
     // there can only be a single PKCS11 provider in
-    // FIPS mode.
+    // restricted security FIPS mode.
```

Comment:
I think "FIPS" shoud be omitted here.

Reply:
@keithc-ca Because the code changes here only apply for the FIPS mode, and the FIPS mode is belong to the restricted security mode, which policy number is 1. So I changed this "FIPS mode" to "restricted security FIPS mode".

Reply:
Unless I misunderstand, FIPS is a special kind of restricted security mode, so saying both is just redundant. If the comment only applies to FIPS mode, then don't change it; if it applies more broadly to any restricted security mode, then mentioning FIPS is misleading. Comments in several other places should be updated accordingly.

Reply:
@keithc-ca Ok, I will remove the "restricted security" and only use the "FIPS" for those FIPS specific changes.